akka-http-tests/src/test/java/akka/http/javadsl/server/values/FormFieldsTest.java

```diff
@@ -98,227 +99,18 @@ public void testByteFormFieldExtraction() {
     }
 
     @Test
-    public void testShortFormFieldExtraction() {
```

Comment:
The type-specific tests were dropped since the new `FormFieldDirectives` only has a single typed `formField` method, that takes an `Unmarshaller[String,T]`. So `BYTE()` above covers that. We can bring them back for consistency of course, if you prefer.

The other unmarshallers in `StringUnmarshallers` are tested in `ParametersTest`.